app/controllers/api/v1/portfolio_items_controller.rb

```diff
@@ -72,7 +72,7 @@ def portfolio_item_params
       end
 
       def portfolio_item_patch_params
-        params.permit(:favorite, :name, :description, :orphan, :state, :display_name, :long_description, :distributor, :documentation_url, :support_url, :workflow_ref, :id, :service_offering_source_ref)
+        params.permit(:favorite, :name, :description, :orphan, :state, :long_description, :distributor, :documentation_url, :support_url, :workflow_ref, :id, :service_offering_source_ref)
```

Comment:
@syncrou now that we have the parameter mixin from the common gem we should be able to use 
https://github.com/ManageIQ/manageiq-api-common/blob/001e9caac08119835e12b9ddc5c8d82b152fce72/lib/manageiq/api/common/application_controller_mixins/parameters.rb#L101
I think we need to include the parameter mixin in our application controller akin to this
https://github.com/ManageIQ/manageiq-api-common/blob/001e9caac08119835e12b9ddc5c8d82b152fce72/spec/dummy/app/controllers/application_controller.rb#L8

Reply:
@mkanoor I actually already did that in #403 , so as long as either that one goes first it'll conflict this or if this one goes first it'll conflict and we'll make it work either way.